movai_core_shared/common/config.py

```diff
@@ -0,0 +1,9 @@
+from movai_core_shared.envvars import REDIS_MASTER_HOST
+
+def is_manager() -> bool:
+    """Tell the process if its running on the manager host machine.
+
+    Returns:
+        bool: true if the fuction runs on a manager host machine, false otherwise.
+    """
+    return REDIS_MASTER_HOST == "redis-master"
```

Comment:
cant it be none?

Reply:
didn't fully understand your question

Reply:
from what I saw movai-service set it.

Reply:
`if REDIS_MASTER_HOST in [None, "redis-master"]`

Reply:
I believe its better to leave it in that case because unsetting this env on a container can cause it to be equal to None, even if it not the actual manager

Reply:
I think this function and the next one (get_ip_address) should be moved under one script and we can call it utilities for example which is more convenient in my opinion